scripts/manage-members.js

```diff
@@ -0,0 +1,177 @@
+'use strict'
+
+const P = require('effd')
+const request = require('request')
+const debug = require('debug')('manage-members')
+const logger = require('../lib/logger.js')
+
+var login
+const github = require('../lib/github-client.js')
+// store/cache info about the authenticated user
+github.users.get({}).then((user) => {
```

Comment:
> But either way- those other tests should only load the scripts they are testing.

I see it from a different angle. Those tests as integration tests, with the intention of ensuring the bot as a whole plays nice together. In that sense loading all the scripts are actually valuable. E.g. a new script may add a route which overrides another script's route, meaning the pre-existing script would suddenly not be triggered at all. We won't be able to detect such issues if we only load one script at a time.

I'm all for running unit tests in complete isolation. Integration tests not so much.